utils/docker_utils.go

```diff
@@ -110,10 +110,8 @@ func pullImageFromRepo(image string) (string, string, error) {
 }
 
 type HistDiff struct {
-	Image1 string
```

Comment:
I pulled them out into the DiffResult instead because they were previously in this Diff field of the DiffResult, and they didn't seem to be part of the actual diff per se